core/whistle_number_manager-1.0.0/include/wh_number_manager.hrl

```diff
@@ -73,7 +73,7 @@
                                        ,<<"failover">> ,<<"prepend">>
                                       ]).
 
--define(WNM_DB_PREFIX_L, "numbers/").
+-define(WNM_DB_PREFIX_L, "numbers/+").
```

Comment:
@lazedo 
Why this change?  

Also, there is no guarantee that this is true.  If the number_manager e164 converters are configured differently on a cluster it may not result in a + prefix.  We have seen this before where a system had purposely been setup to result in US numbers without a '+1' prefix....